go.mod

```diff
@@ -60,56 +60,55 @@ require (
 	github.com/jbenet/go-random v0.0.0-20190219211222-123a90aedc0c
 	github.com/jbenet/go-temp-err-catcher v0.1.0
 	github.com/jbenet/goprocess v0.1.4
-	github.com/libp2p/go-libp2p v0.10.0
-	github.com/libp2p/go-libp2p-circuit v0.2.3
+	github.com/libp2p/go-libp2p v0.10.4-0.20200819182335-c24d028099c9
+	github.com/libp2p/go-libp2p-circuit v0.3.1
 	github.com/libp2p/go-libp2p-connmgr v0.2.4
-	github.com/libp2p/go-libp2p-core v0.6.0
-	github.com/libp2p/go-libp2p-discovery v0.4.0
+	github.com/libp2p/go-libp2p-core v0.6.1
+	github.com/libp2p/go-libp2p-discovery v0.5.0
 	github.com/libp2p/go-libp2p-http v0.1.5
-	github.com/libp2p/go-libp2p-kad-dht v0.8.2
-	github.com/libp2p/go-libp2p-kbucket v0.4.2
+	github.com/libp2p/go-libp2p-kad-dht v0.8.4-0.20200818143235-e788ffca31cb
+	github.com/libp2p/go-libp2p-kbucket v0.4.6
 	github.com/libp2p/go-libp2p-loggables v0.1.0
-	github.com/libp2p/go-libp2p-mplex v0.2.3
+	github.com/libp2p/go-libp2p-mplex v0.2.4
 	github.com/libp2p/go-libp2p-noise v0.1.1
 	github.com/libp2p/go-libp2p-peerstore v0.2.6
-	github.com/libp2p/go-libp2p-pubsub v0.3.1
-	github.com/libp2p/go-libp2p-pubsub-router v0.3.0
+	github.com/libp2p/go-libp2p-pubsub v0.3.4
+	github.com/libp2p/go-libp2p-pubsub-router v0.3.2-0.20200820033122-0d99aeb37422
```

Comment:
Need to get this tiny fix merged + released https://github.com/libp2p/go-libp2p-pubsub-router/pull/80.